pkg/v1/providers/ytt/vendir/cni/_ytt_lib/LICENSE.md

```diff
@@ -3,20 +3,20 @@ Copyright 2020 VMware, Inc.
 
 The Apache 2.0 license (the "License") set forth below applies to all parts of the Tanzu Community Edition project. You may not use this file except in compliance with the License.
 
-Apache License
+Apache License 
+ 
+Version 2.0, January 2004 
+http://www.apache.org/licenses/ 
 
-Version 2.0, January 2004
-http://www.apache.org/licenses/
-
-TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION 
 
 1. Definitions.
 
 "License" shall mean the terms and conditions for use, reproduction,
 and distribution as defined by Sections 1 through 9 of this document.
 
 "Licensor" shall mean the copyright owner or entity authorized by the
-copyright owner that is granting the License.
+copyright owner that is granting the License.  
```

Comment:
nit: I know this came from the vendir-ed source, but perhaps the extraneous newlines and trailing spaces can be fixed at the source, so this doesn't become a recurring thing.

Reply:
Sure, but as of now we can't change this in this PR, because bolt is already merged. Changing that would change the commit. I can do that in subsequent PR.